src/main/kotlin/inc/evil/d17map/controllers/UserController.kt

```diff
@@ -0,0 +1,32 @@
+package inc.evil.d17map.controllers
+
+import inc.evil.d17map.dtos.UserDto
+import inc.evil.d17map.services.ReservationService
+import inc.evil.d17map.services.UserService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/users")
+class UserController(
+    private val userService: UserService,
+    private val reservationService: ReservationService
+) {
+    @GetMapping("/{id}/future-reservations")
+    fun getUserFutureReservations(@PathVariable("id") id: UUID): ResponseEntity<Any> {
```

Comment:
W sumie skoro wołamy tu ReservationService, to może lepiej przenieść tę metodę jednak do ReservationSerice?